src/Umbraco.Web.UI.Client/src/common/directives/validation/valtab.directive.js

```diff
@@ -31,8 +31,17 @@ function valTab() {
                     scope.tabHasError = false;
                 }
             });
+            var tabShownFunc = function (e) {
+                var tabContent = element.closest(".umb-panel").find("#" + tabId);
+                eventsService.emit('valTab.tabShown', { originalEvent: e, tab: scope.tab, content: tabContent });
```

Comment:
This logic should not exist here. This directive is purely to do with helping out with validation. There is already an event for switching tabs which is controlled by the tabs themselves in `umbraco.directives.directive:umbTabs` and the event is `app.tabChange`. 

I'll remove this and change the logic for this PR.

Reply:
Fixed in rev 9ae4d1cdf50625cd7c55f6fdb917b6dc45f253f4